src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -1343,14 +1454,7 @@ internal static new TextSpan GetDiagnosticSpan(SyntaxNode node, EditKind editKin
 
                 case SyntaxKind.AttributeList:
                     var attributeList = (AttributeListSyntax)node;
-                    if (editKind == EditKind.Update)
-                    {
-                        return (attributeList.Target != null) ? attributeList.Target.Span : attributeList.Span;
-                    }
-                    else
-                    {
-                        return attributeList.Span;
-                    }
+                    return attributeList.Span;
```

Comment:
This change is because otherwise adding `[return: A]` to a local function would just squiggle `return:` which didn't seem nice, so now this method makes no assumptions, and the caller can dictate what gets squiggled.